README.md

```diff
@@ -117,6 +117,33 @@ $ helm install spark-kubernetes-operator \
 https://nightlies.apache.org/spark/charts/spark-kubernetes-operator-0.1.0-SNAPSHOT.tgz
 ```
 
+## Clean Up
+
+Check the existing Spark applications and clusters. If exists, delete them.
+
+```
+$ kubectl get sparkapp
+No resources found in default namespace.
+
+$ kubectl get sparkcluster
+No resources found in default namespace.
+```
+
+Remove HelmChart and CRDs.
+
+```
+$ helm uninstall spark-kubernetes-operator
```

Comment:
Since `Helm` officially doesn't delete the CRDs, we need to clean up manually to be up-to-date.